### PR TITLE
Implement secondary channel IE printer.

### PR DESCRIPTION
**Pull request by zejn**
Implement secondary channel IE printer according to linked iw code.

### CODE REVIEW

File: libnl/nl80211/iw_scan.py [hunk 1]
```diff
@@ -504,7 +504,14 @@ def get_secchan_offs(type_, data):
     type_ -- corresponding `ieprinters` dictionary key for the instance.
     data -- bytearray data to read.
     """
-    raise NotImplementedError
+    chans = [
```
Comment (Robpol86) [inline libnl/nl80211/iw_scan.py]: This list is already defined as the module variable `ht_secondary_offset` at the top of this file on line 67. Please use that instead.

File: libnl/nl80211/iw_scan.py [hunk 2]
```diff
@@ -504,7 +504,14 @@ def get_secchan_offs(type_, data):
...
+        "no secondary",
+        "above",
+        "[reserved!]",
+        "below"]
+    if data[0] < len(chans):
+        return chans[data[0]]
```
Comment (Robpol86) [inline libnl/nl80211/iw_scan.py]: Looking at the upstream C source this function should return `"{0} ({1})".format(ht_secondary_offset[data[0]], data[0])` for this condition.

**Comment by zejn** [comment]
Hi, I'm commenting just to let you know the pull request is updated with respect to your very sensible comments. (Not sure if updating a pull request makes you get a notification.)

**Comment by Robpol86** [comment]
Thanks!

**Pull request merged**